e2e/test-component/scenarios/embedding-sdk/static-dashboard.cy.spec.tsx

```diff
@@ -135,7 +133,7 @@ describe("scenarios > embedding-sdk > static-dashboard", () => {
 
           cy.findByText("Orders in a dashboard").should("not.exist");
           cy.findByText("Orders").should("not.exist");
-          cy.findByText(rowsTextRegex).should("not.exist");
+          H.tableInteractiveBody().should("not.exist");
```

Comment:
Can you add the same assertion in `e2e/test-component/scenarios/embedding-sdk/interactive-dashboard.cy.spec.tsx` for consistency?